app/views/users/show.html.slim

```diff
@@ -46,6 +46,12 @@
         | :
       .col.row-value
         = @user.email
+    .row
+      .col-auto.row-label
+        = t('.personal_openai_api_key')
+        | :
+      .col.row-value
+        = @user.personal_openai_api_key
```

Comment:
Instead of just showing the API-Key, I would just show the information if a key has been entered. That way it is less likely to unintentionally leak secret information

Reply:
how to do that?

Reply:
see new suggestion